src/components/autocomplete.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import debounce from 'lodash.debounce';
```

Comment:
Need to fix it
<img width="878" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/0da5a8d3-fbb5-4c77-b30e-07682b6435d2">

Reply:
Not resolved